adapter-http/src/main/resources/http.md

```diff
@@ -203,7 +203,7 @@ an op template like `space: {space}`, OR use an inline op field in your op
 template like `space: {(ThreadNumToInteger())}`.
 
 You can use any binding function you want for the space op field. However,
-if you were to assign it something like "space: {{Identity()}}" you would
+if you were to assign it something like "space: {(Identity()}" you would
```

Comment:
`#trivial` I see there is an open paranthesis "`(`" but, don't see the equivalent closing paranthesis "`)`" and just wanted to call your attention here to validate if this is indeed the right format.

Reply:
Thanks, I'll fix this.

Reply:
fixed in current code to be pushed.